### PR TITLE
Add new context api polyfill and migrate paralax

### PR DESCRIPTION
**Pull request by TrySound**
Polyfill is also required until major release with 16.4 restriction.

### CODE REVIEW

File: src/Parallax.js [hunk 1]
```diff
@@ -144,7 +138,7 @@ export default class Parallax extends React.PureComponent {
       >
         {this.state.ready && (
           <div
-            ref={node => this.node}
+            ref={node => (this.content = node)}
```
Comment (TrySound) [inline src/Parallax.js]: Oops, I wasn't careful here.

**Comment by TrySound** [comment]
We need to add tests. Which ones would you prefer? I can help to setup puppeteer.

**Comment by drcmda** [comment]
Sure this needs a polyfill? 16.3.1 contains the new context, already using in a couple of my apps.

Testing would be awesome, no preferences as you seem to know your way around pretty well.

**Comment by drcmda** [comment]
Btw, i'm looking for help with maintenance, sharing authorship. If you're down i'll give you all the access you need to publish/push/pull/merge/review on your own.

**Comment by TrySound** [comment]
The latest major release of this package included 16.0.0 and if we will publish minor release with new api those users can be surprised.

I can partially help with maintenance. However I'm not interested in project itself.

Probably you guys have some interest in it?

/cc @alexreardon @souporserious

**Comment by alexreardon** [comment]
Can you add a little more context @TrySound ?

**Comment by alexreardon** [comment]
This project generally or the parallax effect?

**Comment by TrySound** [comment]
The project. The discussion moved to maintenance question. :)

**Pull request merged**